recipes/shapelib/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "shapelib" %}
+{% set version = "1.4.0" %}
+{% set sha256 = "3eddd5f2690f2cf18d8d9798267537a2102a289020b9818f9a31eb626f91a7a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://download.osgeo.org/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+    - cmake
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
If you add the `vc` dependencies at `build` and `run` above you don't need this `hack`.